### DISCUSSION TITLE
Is this alive?  Any other forum/user-group?

### DISCUSSION THREAD

**Discussion by barajasfernando** [pH7Software/pH7-Social-Dating-CMS#934] [General]
Hi!  Is this project alive?  Any other forum or user group?

Thanks!

**Comment by MICNeuromancer** [comment]
it comes sometimes a update... but for my discussions im waiting now nearly a month for a answer... here is no active community...

**Reply by barajasfernando** [reply]
I think this project is dead, then :(

**Reply by MICNeuromancer** [reply]
Not really... only a active community is missing... Pierre can't do everything alone and I think that's Pierre works on other projects now...

**Reply by hobb7t** [reply]
well guys, since nobody did it so far, i took the moment to start a Discord server, my advice is to join this one and build up a more "active" community, i hope that Pierre will be happy to join us too.
Here is the link https://discord.gg/p59BhR3AHe

**Reply by rthuey2** [reply]
joined

**Comment by coolyou11** [comment]
This project needs a proper support online forum, this script is one of the best that I have come across, but it lacks a support forum like phpBB.

**Reply by kromapi** [reply]
Hey folks, just checked out [fart fetish website](https://www.fetfantazy.com/fart.html) — wow, this fart fetish site is definitely not your everyday dating platform. It prides itself on embracing the more unusual attractions with a heavy focus on gas fetish connections. The layout is clean, it feels like the community wants consent and safety to be front and center. If you’re into kink sites that don’t shy away from the weird but are still respectful, this might hit your niche. Not for vanilla dates, but for something more specific, it seems legit.

**Comment by rthuey2** [comment]
So you have a site up and running?Sent from my iThingyOn Sep 29, 2025, at 1:10 AM, kromapi ***@***.***> wrote:﻿
live
